File: dotnet/WebService/src/WebApi/Utils/Base64.cs
using System;
using System.Text;

namespace WebApi.Utils
{
    public static class Base64
    {
        public static string Encode(string message)
        {
            var messageBytes = Encoding.UTF8.GetBytes(message);

            return Convert.ToBase64String(messageBytes);
        }

        public static string Decode(string base64EncodedMessage)
        {
            var base64EncodedBytes = Convert.FromBase64String(base64EncodedMessage);

            return Encoding.UTF8.GetString(base64EncodedBytes);
        }
    }
}

File: laravel/webservice/app/Utils/Base64.php
<?php

namespace App\Utils;

class Base64
{
    public static function encode(string $message): string
    {
        return base64_encode($message);
    }

    public static function decode(string $base64EncodedMessage): string
    {
        return base64_decode($base64EncodedMessage);
    }
}
